tests/test_pages_description.py

```diff
@@ -0,0 +1,170 @@
+import pytest
+from web_poet import HttpResponse, field
+
+from zyte_common_items import ProductPage
+
+HTML = b"""
+<!DOCTYPE html>
+<html>
+    <body>
+        <article><p>text</p><p>text2</p></article>
```

Comment:
What do you think about changing it, so that it's not the same as the cleaned version - e.g. use divs instead of p elements?